cmd/process.go

```diff
@@ -42,6 +42,27 @@ func Start(a *AppState) *cobra.Command {
 			logger := a.Logger
 			cfg := a.Config
 
+			flushInterval, err := cmd.Flags().GetDuration(flagFlushInterval)
+			if err != nil {
+				logger.Error("Invalid flush interval", "error", err)
```

Comment:
should we return here if we encounter an error? users may expect some flush interval to be being used but due to an error they end up with `flushInterval` being initialized to zero.

Reply:
I don't think we'll want to return here. If we always returned here, the relayer would have to run with the flushing mechanism. By throwing an error, we still allow the relayer to run without the flushing mechanism in place.

Reply:
This is at least useful for testing purposes.